tools/ci.sh

```diff
@@ -648,3 +648,20 @@ function ci_zephyr_build {
     docker exec zephyr-ci west build -p auto -b mimxrt1050_evk
     docker exec zephyr-ci west build -p auto -b nucleo_wb55rg # for bluetooth
 }
+
+
+########################################################################################
```

Comment:
please move this new section to alphabetical order, between `ports/qemu-arm` and `ports/rp2`

Reply:
OK, I will move this.

Reply:
done.